javascript/javascript1/week4/week4-homework/getcommand.js

```diff
@@ -0,0 +1,133 @@
+"use strict"
+
+
+
+let names = [];
+let whoAmI = 'What is my name';
+let day = 'What day is it today'
+let todo =[];
+let myTodoList = 'What is on my todo'
+let favDish = [];
+let event = [];
+let eventList = 'What am I doing this week'
+
+
+function getReply(command) {
+    
+    //Saving the person's name and returning nice to meet you plus the person's name
+    if (command.startsWith('Hello') && command.includes('name')) {
+          let commandArray = command.split(' ')
+          let name = commandArray[commandArray.length -1]
+          names.push(name)
+        
+        return `Nice to meet you ${names[names.length -1]}`;
```

Comment:
```suggestion
        return `Nice to meet you ${name}`;
```
No need to access the array again :) ...One of the most important things you need to remember in software development is not to repeat yourself (google *DRY principle* if you want to learn more about it :) )

Reply:
okay.